recipes/proteoformclassifier/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "ProteoformClassifier" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/smith-chem-wisc/ProteoformClassifier/releases/download/{{ version }}/CMD.zip
+  sha256: 7d67f929fddd26835968a436779fcc8b7210dc3c3807261a36056a5be9fa49fc
+
+build:
+  skip: true  # [linux]
```

Comment:
`dotnet-runtime` requires cos7 currently, so this will be enabled upon making the feedstock after specifying that in the conda-forge.yml file like so: https://github.com/conda-forge/metamorpheus-feedstock/blob/master/conda-forge.yml#L3